draft-xyz-rats-psa-endorsements.md

```diff
@@ -81,32 +88,35 @@ The reader is assumed to be familiar with the terms defined in Section 2.1 of
 {: #sec-psa-endorsements }
 
 PSA Endorsements describe an attesting device in terms of the hardware and
-firmware components that make up its PSA Root of Trust (RoT).  This includes
+firmware components that make up its PSA Root of Trust (RoT). This includes
 the identification and expected state of the device as well as the
 cryptographic key material needed to verify Evidence signed by the device's PSA
-RoT.  Additionally, PSA Endorsements can include information related to the
+RoT. Additionally, PSA Endorsements can include information related to the
 certification status of the attesting device.
 
-There are two basic types of PSA endorsements:
+There are three basic types of PSA endorsements:
 
 * Reference Values ({{sec-ref-values}}), i.e., measurements of the PSA RoT
   firmware;
 * Identity Claims ({{sec-identity}}), i.e., cryptographic keys that can be used
-  to verify signed Evidence produced by the PSA RoT.
+  to verify signed Evidence produced by the PSA RoT and the associated identifiers that
+  bind the keys to their device instances.
+* Certification Claims ({{sec-certificates}}), i.e., metadata that describe
+  the certification status associated with a PSA device.
 
 ## Reference Values
 {: #sec-ref-values}
 
 Reference Values carry measurements and other metadata associated with the
-updatable firmware in a PSA RoT.  When appraising Evidence, the Verifier
+updatable components in a PSA RoT.  When appraising Evidence, the Verifier
```

Comment:
we can change it to elements!

Reply:
I don't understand what is wrong with the original text?  Why is firmware wrong and need to be changed?

Reply:
technically update component can be anything right, not just firmware,

Reply:
My only intent is to clarify precisely what we mean.  Clearly you think that the term firmware is too limited, but what else do we allow updating?  We should be able to give that "something" (is it configuration? or something else?) a name and be precise instead of using a wildcard term.  Otherwise let's stick to firmware.

Reply:
ok, I will change this!